### PR TITLE
Updated Text Changes

### PR DESCRIPTION
**Pull request by klk1236**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ componentDidMount() {
             <Route path="/team" component={Team} />
             <Route path="/roadmap" component={Roadmap} />
             <Route path="/app" component={AppContainer} />
-            <Route path="/stakedrop" component={stakedrop} />
+            <Route path="/StakeDrop" component={stakedrop} />
```
Comment (deepanshutr) [inline src/App.js:35]: pl check if correct

Comment (klk1236) [reply src/App.js:35]: Yes its not a good practice to use in URL Capital letters, But in our website all campaign URL's like this only, I dont't know exactly someone suggested to raju to use like this in URL. If you suggest me to remove that means i do that all URL's.

Comment (deepanshutr) [reply src/App.js:35]: got you

**Review by deepanshutr — APPROVED** [review]
Reviewed.

**Pull request merged**